docs/rewards/reward-boosts.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: reward-boosts
+title: Rewards Boosts
+sidebar_label: Rewards Boosts
+slug: /project/rewards-boosts
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+## Overview
+In addition to the **Base Rewards**, a device already gets for contributing to weather data during the regular daily distribution. Learn more about the daily rewards [Reward Mechanism (RM) ](/reward-mechanism).
+
+**Reward Boosts** rewards devices with extra $WXM tokens for meeting specific criteria. 
+
+So far the following **Reward Boosts** are implemented:
```

Comment:
Hmmm not true, Beta Rewards is almost implemented, but Cell Boost is **not**, it's just the next boost down in the pipeline. So let's not say "implemented" here as it's a wrong term.